AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.ts

```diff
@@ -197,52 +159,8 @@ export class AddParticipantComponent extends BookingBaseComponent implements OnI
     }
 
     initializeForm() {
-        this.role = new FormControl(this.constants.PleaseSelect, [
-            Validators.required,
-            Validators.pattern(this.constants.PleaseSelectPattern)
-        ]);
-        this.party = new FormControl(this.constants.PleaseSelect, [
-            Validators.required,
-            Validators.pattern(this.constants.PleaseSelectPattern)
-        ]);
-        this.title = new FormControl(this.constants.PleaseSelect);
-        this.firstName = new FormControl('', [
-            Validators.required,
-            Validators.pattern(Constants.TextInputPatternName),
-            Validators.maxLength(255)
-        ]);
-        this.lastName = new FormControl('', [
-            Validators.required,
-            Validators.pattern(Constants.TextInputPatternName),
-            Validators.maxLength(255)
-        ]);
-        this.phone = new FormControl('', [Validators.required, Validators.pattern(Constants.PhonePattern)]);
-        this.displayName = new FormControl('', [
-            Validators.required,
-            Validators.pattern(Constants.TextInputPatternName),
-            Validators.maxLength(255)
-        ]);
-        this.companyName = new FormControl('');
-        this.companyNameIndividual = new FormControl('', [Validators.pattern(Constants.TextInputPattern), Validators.maxLength(255)]);
-        this.representing = new FormControl('', [Validators.pattern(Constants.TextInputPattern), Validators.maxLength(255)]);
-        this.interpreterFor = new FormControl(this.constants.PleaseSelect, [
-            Validators.required,
-            Validators.pattern(this.constants.PleaseSelectPattern)
-        ]);
-
-        this.form = new FormGroup({
-            role: this.role,
-            party: this.party,
-            title: this.title,
-            firstName: this.firstName,
-            lastName: this.lastName,
-            phone: this.phone,
-            displayName: this.displayName,
-            companyName: this.companyName,
-            companyNameIndividual: this.companyNameIndividual,
-            representing: this.representing,
-            interpreterFor: this.interpreterFor
-        });
+        this.initialiseForm();
+        this.party.setValidators([Validators.required, Validators.pattern(this.constants.PleaseSelectPattern)]);
```

Comment:
Why are all the others moved but part validators is here?

Reply:
There are quite a few fields which should actually belong in the AddParticipantComponent instead of the base class, such as the party field. But they were left inside the base as they are used when the email search component emits the found participant. 

Basically the refactoring would have taken a lot longer do it 100% so it's been I'd say 80%. Because of the time pressure to get this out, another story has been created as tech debt to refactor this completely.

This might be confusing so can talk about it on a call if needed.

Reply:
No I get it after going over some more of the code. And yeah if there's tech debt and a ticket for it that's fine for me.